### PR TITLE
Default Space Images

### PR DESCRIPTION
**Pull request by kobergj**
Moves space creation logic (`.space` folder, initial `readme.md`, default image) to server instead relying on client logic.

~Needs https://github.com/cs3org/reva/pull/4545~
~Needs web changes (less requests)~

Adds a new parameter "template" to the create drive request. Setting this to "default" will auto generate `.space` folder, `readme.md` and a space image.

The CS3 metadata client using service accounts talks directly to the storage providers. 

Can be extended later to support custom space templates.

### CODE REVIEW

**Comment by TheOneRing** [comment]
Thank you!!

**Comment by tbsbdr** [comment]
The Space Icon in Web is: https://remixicon.com/icon/function-fill

File: services/graph/pkg/service/v0/utils.go [hunk 1]
```diff
@@ -425,3 +433,104 @@ func cs3ReceivedShareToLibreGraphPermissions(ctx context.Context, logger *log.Lo
 
 	return permission, nil
 }
+
+func (g Graph) applySpaceTemplate(gwc gateway.GatewayAPIClient, root *storageprovider.ResourceId, template string) (*v1beta1.Opaque, error) {
+	var fsys fs.ReadDirFS
+
+	switch template {
```
Comment (2403905) [inline services/graph/pkg/service/v0/utils.go:440]: It is just `if else` 
re we going to extend this in the future?

Comment (kobergj) [reply services/graph/pkg/service/v0/utils.go:440]: For now it has only one option, yes. So could change it to a if clause if you think that is simpler. However we are planning to extend the templates with cutom options later. 

Not sure what is better for now. Would you prefer an if clause?

Comment (2403905) [reply services/graph/pkg/service/v0/utils.go:440]: Let's keep it like as is

**Review by 2403905 — APPROVED** [review]

**Comment by kobergj** [comment]
Rebased to master because of merge conflicts - no code changes

File: services/graph/pkg/service/v0/spacetemplate/.space/readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Here you can add a description for this Space.
```
Comment (kulmann) [inline services/graph/pkg/service/v0/spacetemplate/.space/readme.md:1]: Web was putting a translated text, based on the language of the user. Can you translate the text backend side as well? `Accept-language` header should be available on the `POST` request that creates the space.

Comment (AlexAndBear) [reply services/graph/pkg/service/v0/spacetemplate/.space/readme.md:1]: That would be a hard requirement to enable this feature 🙋‍♀️

Comment (kobergj) [reply services/graph/pkg/service/v0/spacetemplate/.space/readme.md:1]: Collecting issues with space templates here: https://github.com/owncloud/ocis/issues/8713

**Pull request merged**